### PR TITLE
fusion_volume: add option to copy content of a volume or a volume snapshot to a new or already existing volume

### PR DESCRIPTION
**Pull request by andreepyro**
##### BLOCKED
This Pull Request introduces a feature which isn't part of the purefusion Python SDK yet. When it is released, this PR can be merged.

##### SUMMARY
Pure Storage Fusion is going to allow users to create/patch a volume with data from another volume or volume snapshot. In order to provide Ansible users the same set of features as the purefusion Python SDK has, we have to introduce this feature for out Ansbile Collection too.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
plugins/fusion_volume.py

### CODE REVIEW

File: requirements.txt
Comment (andreepyro) [inline requirements.txt:1]: **Don't forget to apply** and **to correct** this suggestion before merging this Pull Request!
```suggestion
purefusion>=1.1.0,<2.0
```
The feature introduced in this PR isn't backward compatible with old versions of purefusion Python SDK. Forward compatibility remains.

**Comment by andreepyro** [comment]
Btw, what version of `pylint` do we use? It throws a warning that should be [already fixed](https://github.com/PyCQA/pylint/pull/4064).

**Comment by andreepyro** [comment]
Also, I suggest writing out versions of all sanity checking tools we use at the beginning of our CI, so we can debug easier.

**Comment by sdodsley** [comment]
> Btw, what version of `pylint` do we use? It throws a warning that should be [already fixed](https://github.com/PyCQA/pylint/pull/4064).

Ansible core team control that I suspect in their test image

File: requirements.txt
Comment (andreepyro) [reply requirements.txt:1]: It's `1.0.11` in the end.

**Review by sdodsley — DISMISSED** [review]

**Review by genalt — DISMISSED** [review]
LGTM

**Pull request merged**
